conf/messages

```diff
@@ -294,6 +294,15 @@ taskList.agent.employmentTask = PAYE employment (including student loans)
 taskList.cisTask = Construction Industry Schemes deductions
 taskList.agent.cisTask = Construction Industry Schemes deductions
 
+taskList.checkSEDetails = Check your self-employment details
+taskList.agent.checkSEDetails = Check your self-employment details
+
+taskList.industrySector = Industry sectors and working abroad
+taskList.agent.industrySector = Industry sectors and working abroad
+
+taskList.yourIncome = Your income
+taskList.agent.yourIncome = Your income
+
```

Comment:
Do the agent messages need to be different?

Reply:
Good spot, will amend